pkg/apis/extensions/v1beta1/types.go

```diff
@@ -968,27 +968,27 @@ type HostPortRange struct {
 	Max int32 `json:"max"`
 }
 
-// SELinux Context Strategy Options defines the strategy type and any options used to create the strategy.
-type SELinuxContextStrategyOptions struct {
+// SELinux Strategy Options defines the strategy type and any options used to create the strategy.
+type SELinuxStrategyOptions struct {
 	// type is the strategy that will dictate the allowable labels that may be set.
-	Type SELinuxContextStrategy `json:"type"`
-	// seLinuxOptions required to run as; required for MustRunAs
+	Type SELinuxStrategy `json:"type"`
+	// Options required to run as; required for MustRunAs
 	// More info: http://releases.k8s.io/HEAD/docs/design/security_context.md#security-context
-	SELinuxOptions *v1.SELinuxOptions `json:"seLinuxOptions,omitempty"`
```

Comment:
I'm trying to follow the go "don't stutter" rule.  If the type already says
"selinux" and the JSON path to the field says "selinux", it's nice if we
can not have to say "selinux" again.  Not a huge deal, certainly not
something I am going to fight over.

On Thu, Mar 3, 2016 at 7:31 AM, Eric Tune <notifications@github.com> wrote:

> In pkg/apis/extensions/v1beta1/types.go
> <https://github.com/kubernetes/kubernetes/pull/22419#discussion_r54895519>
> :
>
> >  	// More info: http://releases.k8s.io/HEAD/docs/design/security_context.md#security-context
> > -	SELinuxOptions *v1.SELinuxOptions `json:"seLinuxOptions,omitempty"`
>
> this is called seLinuxOptions in the container[].securityContext. Do you
> like shortness in one place or consistency in two places?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/22419/files#r54895519>.
>